lua/wire/zvm/zvm_core.lua

```diff
@@ -283,22 +283,24 @@ function ZVM:Dyn_EmitInterruptCheck()
         self:Dyn_EmitBreak()
       self:Emit("elseif VM.MEMRQ == 2 then") -- Reading
         self:Dyn_EmitState(true)
+		self:Emit("VM.MEMRQ = 4")
         self:Emit("VM.IP = "..self.PrecompileStartIP)
         self:Emit("VM.XEIP = "..(self.PrecompileTrueXEIP or 0))
-        self:Emit("VM:Interrupt(29,0)")
+        self:Emit("VM:Interrupt(28,VM.LADD)")
         self:Dyn_EmitBreak()
       self:Emit("elseif VM.MEMRQ == 3 then") -- Writing
         self:Dyn_EmitState(true)
+		self:Emit("VM.MEMRQ = 5")
         self:Emit("VM.IP = "..self.PrecompileStartIP)
         self:Emit("VM.XEIP = "..(self.PrecompileTrueXEIP or 0))
-        self:Emit("VM:Interrupt(30,VM.LADD)")
-        self:Dyn_EmitBreak()
+        self:Emit("VM:Interrupt(29,VM.LADD)")
+		self:Dyn_EmitBreak()
       self:Emit("end")
     self:Emit("end")
   end
   self:Emit("if VM.INTR == 1 then")
-    self:Dyn_EmitState(true)
-    self:Dyn_EmitBreak(true)
+    --self:Dyn_EmitState(true)
```

Comment:
It's okay to just remove the old lines - if you need them back later, just look through the Git history before you made the change. That's what Git is for :wink: